list.json

```diff
@@ -1,146 +1,286 @@
 			"parts": 1
 		},
 		"20-newsgroups": {
-			"description": "The 20 Newsgroups data set is a collection of approximately 20,000 newsgroup documents, partitioned (nearly) evenly across 20 different newsgroups",
+			"num_rows": 18846,
+			"row_format": "dict",
+			"size": "14483581 (~14MB)",
+			"reader": "https://github.com/RaRe-Technologies/gensim-data/releases/download/20-newsgroups/__init__.py",
+			"license": "not found",
+			"fields": {
+				"topic": "name of topic (20 variant of possible values)",
+				"set": "marker of original split (possible values 'train' and 'test')",
+				"data": "",
+				"id": "original id inferred from folder name"
+			},
+			"description": "Collection of approximately 20,000 newsgroup documents, partitioned (nearly) evenly across 20 different newsgroups.",
 			"checksum": "c92fd4f6640a86d5ba89eaad818a9891",
 			"file_name": "20-newsgroups.gz",
-			"source": "http://qwone.com/~jason/20Newsgroups/",
+			"links": ["http://qwone.com/~jason/20Newsgroups/"],
 			"parts": 1
 		},
 		"__testing_matrix-synopsis": {
-			"description": "Synopsis of the movie matrix",
+			"description": "[THIS IS ONLY FOR TESTING] Synopsis of the movie matrix.",
 			"checksum": "1767ac93a089b43899d54944b07d9dc5",
 			"file_name": "__testing_matrix-synopsis.gz",
-			"source": "http://www.imdb.com/title/tt0133093/plotsummary?ref_=ttpl_pl_syn#synopsis",
+			"links": ["http://www.imdb.com/title/tt0133093/plotsummary?ref_=ttpl_pl_syn#synopsis"],
 			"parts": 1
 		},
 		"__testing_multipart-matrix-synopsis": {
-			"description": "Synopsis of the movie matrix",
+			"description": "[THIS IS ONLY FOR TESTING] Synopsis of the movie matrix.",
 			"checksum-0": "c8b0c7d8cf562b1b632c262a173ac338",
 			"checksum-1": "5ff7fc6818e9a5d9bc1cf12c35ed8b96",
 			"checksum-2": "966db9d274d125beaac7987202076cba",
 			"file_name": "__testing_multipart-matrix-synopsis.gz",
-			"source": "http://www.imdb.com/title/tt0133093/plotsummary?ref_=ttpl_pl_syn#synopsis",
+			"links": ["http://www.imdb.com/title/tt0133093/plotsummary?ref_=ttpl_pl_syn#synopsis"],
 			"parts": 3
 		}
 	},
 	"models": {
 		"word2vec-google-news-300": {
-			"description": "Pre-trained vectors trained on part of Google News dataset (about 100 billion words). The model contains 300-dimensional vectors for 3 million words and phrases. The phrases were obtained using a simple data-driven approach described in 'Distributed Representations of Words and Phrases and their Compositionality', https://code.google.com/archive/p/word2vec/",
-			"parameters": "dimension = 300",
-			"papers": "https://arxiv.org/abs/1301.3781, https://arxiv.org/abs/1310.4546, https://www.microsoft.com/en-us/research/publication/linguistic-regularities-in-continuous-space-word-representations/?from=http%3A%2F%2Fresearch.microsoft.com%2Fpubs%2F189726%2Frvecs.pdf",
+			"type": "word2vec",
+			"num_vectors": 3000000,
+			"size": "1743563840 (~1.7GB)",
+			"base_dataset": "Google News (about 100 billion words)",
```

Comment:
Do we really need this field? Belongs in the description IMO.

What we really want from the metadata:

* dataset **purpose** (its description, motivation, intended use-cases) and overall stats (size in bytes, number of records)
* the dataset format (how to use the dataset = concrete "reader")
* the process used in creating this dataset (such as number of dimensions, preprocessing, original data, original articles…)
* license for data redistribution (for us) and for its use (for users who download the data)

Reply:
Yes, because this gives at least some idea of the model (domain field, how large souce corpus, etc).
Unfortunately about such datasets either there is absolutely no information, or only such a brief description (but it's better than nothing).